aglu.md

```diff
@@ -197,6 +196,25 @@ Consider the example of a future scenario with a high global demand for a bioene
 
 Although GCAM is not structured as an optimization model, the allocation of production of crops and products across regions and subregions of the globe based on comparative advantage can be considered optimal in terms of maximizing global profits (which is not the same as minimizing land requirements). While each land subregion makes its own independent allocation, the subregions communicate these decisions with each other through economic markets. The global markets for agricultural and forest products react to these allocations by comparing global supplies to demands and adjusting prices to equilibrate supplies and demands. The resulting prices are sent back to each land subregion as signals as to how its land allocation should be changed.  The process of allocation and price adjustment continues until the markets are in equilibrium.  This market equilibrium is an economically efficient allocation of land resources among regions (Samuelson and Nordhaus, 1985).
 
+### Regional Agricultural Markets and Trade
+Version 5.1.3 and greater of GCAM include the capability to represent regional agricultural markets, with explicitly modeled gross imports and exports of selected crop types. In contrast to the purely global approach, wherein crop prices are only tracked at the global level, the regional markets approach also has region-specific crop prices and markets. Note that bilateral trade is not modeled; for each crop, a region's gross imports come from, and exports go to, a single global market for the given crop.
```

Comment:
yes I think so.   But I don't want to get the definition wrong.  t came in in June 2019, when we were on version 5.1.3.

Reply:
The release version of 5.1.3 didn't include it so I would advocate just bumping it to `5.2` to avoid confusion.  I can just make the change if we are in agreement.

Reply:
Wait, but it wasn't in Release 5.1.3 which was in February.  So, we should probably say starting with Release 5.2.

Reply:
Yes we are on the same Page literally and conceptually.

Reply:
Fixed